salesforce_bulk/salesforce_bulk.py

```diff
@@ -172,18 +172,27 @@ def create_delete_job(self, object_name, **kwargs):
         return self.create_job(object_name, "delete", **kwargs)
 
     def create_job(self, object_name=None, operation=None, contentType='CSV',
-                   concurrency=None, external_id_name=None):
+                   concurrency=None, external_id_name=None, pk_chunking=False):
         assert(object_name is not None)
         assert(operation is not None)
 
+        extra_headers = {}
+        if pk_chunking:
+            if pk_chunking is True:
+                pk_chunking = u'true'
+            elif isinstance(pk_chunking, int):
+                pk_chunking = u'chunkSize=%d;' % pk_chunking
+
```

Comment:
Should we have an `else pk_chunking = False` to guard against it being something other than a bool or int? Either that or add an assert.

Reply:
There's a if pk_chunking that guards against that. I'm calling 0, None, False all off here basically. I could wrap with str/unicode to ensure okay types I guess.